main.go

```diff
@@ -31,22 +31,27 @@ const (
 	diskPathConverted    string = "./tmp/disk.qcow2"
 )
 
-func applyVirtualMachineExport(vmName string) error {
+func applyVirtualMachineExport(vmNamespace, vmName string) error {
 	log.Println("Applying VirtualMachineExport object...")
 
 	client, err := kubecli.GetKubevirtClient()
 	if err != nil {
 		return err
 	}
 
-	vmNamespace := os.Getenv("VM_NAMESPACE")
+	env := os.Getenv("VM_NAMESPACE")
+	if env != "" {
+		vmNamespace = env
+	}
+
 	if vmNamespace == "" {
-		return fmt.Errorf("VM namespace is not defined. Set VM_NAMESPACE.")
+		return fmt.Errorf("VM namespace is not defined. Set VM_NAMESPACE or parameter.")
 	}
 
 	vmExport := &v1beta1.VirtualMachineExport{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: vmName,
+			Name:      vmName,
```

Comment:
if the user passes some wrong combination of the name / namespace, will the virtctl exit immediatelly or will it wait if the vm shows up?